### PR TITLE
Implemented Record

### PR DESCRIPTION
**Pull request by sonufrienko**
Added feature to transmit microphone stream to the SPICE.

### CODE REVIEW

File: src/record.js [hunk 1]
```diff
@@ -41,19 +40,104 @@ function SpiceRecordConn()
...
+
+        this.send_start_mark_message();
+        this.send_mode_message();
+        this.start_microphone();
     }
 
     if (msg.type == Constants.SPICE_MSG_RECORD_STOP)
     {
         Utils.DEBUG > 0 && console.log("RECORD_STOP");
+        debugger;
```
Comment (jwhite66) [inline src/record.js]: This should not be in any patch.

Comment (sonufrienko) [reply src/record.js]: @jwhite66 Thanks.
:eyeglasses: Fixed

**Comment by jwhite66** [comment]
This looks like great work.  But a few of my comments still apply; this is not really the right venue for reviewing patches, you should have your copyright, and I'd like to see the patches consolidated.  Since they're new, we don't really need to see your development history.

**Comment by freddy77** [comment]
Any update on this? Any reason why is stuck? Maybe the author is not able to dedicate time and we should pick up the job (we would need author to agree) ?

**Comment by fdo-mirror** [comment]
freedesktop doesn't allow pull requests on its mirrors. This is an automated message.

**Pull request closed without merging**